lessons/pythonClasses/lesson.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Python Classes
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Introduction to Classes in Python
+
+ - **Authors**: Lina Tran
+ - **Research field**: Neuroscience
+ - **Lesson topic**: Object Oriented Programming with Python
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/pythonClasses>
+
+## Topics Covered:
+- Defining a Class
+- Creating Instances of a Class
+- Attributes
+ + Class Attributes vs. Normal Attributes
```

Comment:
The subitems in your lists won't work on the website. Any subitems must have two leading spaces